calendar/screen.css

```
@@ -92,20 +93,165 @@ body {
+}
+.toastui-calendar-popup-arrow.toastui-calendar-left .toastui-calendar-popup-arrow-fill {
+  border-right-color: var(--grist-theme-popup-bg);
+}
+.toastui-calendar-popup-arrow.toastui-calendar-top .toastui-calendar-popup-arrow-border {
+  border-bottom-color: var(--grist-theme-modal-border-dark);
+}
+.toastui-calendar-popup-arrow.toastui-calendar-top .toastui-calendar-popup-arrow-fill {
+  border-bottom-color: var(--grist-theme-popup-bg);
+}
+.toastui-calendar-popup-arrow.toastui-calendar-bottom .toastui-calendar-popup-arrow-border {
+  border-top-color: var(--grist-theme-modal-border-dark);
 }
+.toastui-calendar-popup-arrow.toastui-calendar-bottom .toastui-calendar-popup-arrow-fill {
+  border-top-color: var(--grist-theme-popup-bg);
+}
+.toastui-calendar-popup-button.toastui-calendar-popup-confirm {
+  /* match grist primary button style */
+  background-color: var(--grist-theme-control-primary-bg);
+  color: var(--grist-theme-control-primary-fg);
+  border: none;
+  border-radius: 4px;
+  margin: 4px 0;
+  font-weight: 500;
+  font-size: 13px;
+  padding: 10px 24px;
+  min-height: 40px;
+  height: unset;
+  width: unset;
+}
+.toastui-calendar-popup-button.toastui-calendar-popup-close {
+  background-color: transparent;
+  padding: 4px;
+  margin: -4px;
+  font-style: normal;
+  font-size: 16px;
+  color: var(--grist-theme-modal-close-button-fg);
+  width: 24px;
+  height: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+}
+.toastui-calendar-popup-button.toastui-calendar-popup-close:hover {
+  background-color: var(--grist-theme-hover);
+}
+.toastui-calendar-popup-button.toastui-calendar-popup-close::before {
+  content: "✕";
+}
+.toastui-calendar-icon.toastui-calendar-ic-close {
+  display: none;
+}
+
+.toastui-calendar-popup-section-allday .toastui-calendar-ic-checkbox-normal,
+.toastui-calendar-icon.toastui-calendar-ic-checkbox-checked {
+  background: none;
+  position: relative;
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+}
+.toastui-calendar-popup-section-allday .toastui-calendar-ic-checkbox-normal::after {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 16px;
+  width: 16px;
+  border: 1px solid var(--grist-theme-checkbox-border);
+  border-radius: 3px;
+}
+.toastui-calendar-icon.toastui-calendar-ic-checkbox-checked {
+  background-color: var(--grist-theme-control-primary-bg);
+  border-radius: 3px;
+}
+.toastui-calendar-icon.toastui-calendar-ic-checkbox-checked::after {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 16px;
+  width: 16px;
+  -webkit-mask-image: var(--icon-Tick);
+  -webkit-mask-size: contain;
+  -webkit-mask-position: center;
+  -webkit-mask-repeat: no-repeat;
+  background-color: var(--grist-theme-control-primary-fg);
+}
+.toastui-calendar-edit-button .toastui-calendar-icon,
+.toastui-calendar-delete-button .toastui-calendar-icon {
+  display: none;    /* no easy way to adjust color for dark mode */
+}
+
```

```suggestion

.toastui-calendar-detail-container .toastui-calendar-section-button {
  border-top: 1px solid var(--grist-theme-menu-border);
}

```
This is the border that's visible above the buttons in the edit popup.